spec/commands/order/add_guest_spec.rb

```diff
@@ -1,12 +1,14 @@
 require 'commands/order/add_guest'
+require 'models/order'
+require 'spec_helper'
 
 RSpec.describe AddGuest do
   let (:event_data_from_will) { {user_id: "asdf", user_name: "Will" } }
 
   it "save a guest" do
     response = add_guest("james smith")
 
-     expect(Order.count).to eq(1)
+     expect(Order.last(:user_name => "james smith")).not_to be(nil)
```

Comment:
Those two expectations verify two different and non-related things. I think it's worth splitting them. One test then will verify that the guest entry has been created in the database. The second test can verify that `run()` returns the correct string.